_layouts/default.html

```diff
@@ -168,9 +168,9 @@
         <!-- /.container -->
       </div>
     </div>
-    <script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.min.js"></script>
-    <script src="js/algolia.js"></script>
+    <script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js" defer></script>
```

Comment:
`defer` has no effect here because it’s the end of the file—please remove here (all three occurrences).

See e.g. https://pagespeedchecklist.com/async-and-defer for details.